### PR TITLE
Hawq 1385. fixed hawq_ctl stop failed when master is down

### PR DESCRIPTION
**Pull request by interma**
Get hawq_acl_type from hawq_site.xml if connect db failed.

### CODE REVIEW

**Comment by interma** [comment]
@linwen @wcl14 @ictmalili help to review, tks!

File: tools/bin/hawq_ctl [hunk 1]
```diff
@@ -835,6 +835,8 @@ class HawqStop:
                 acl_type = rows.next()[1]
             conn.close()
         except DatabaseError, ex:
+            # get hawq_acl_type from hawq_site.xml if connect db failed.
```
Comment (linwen) [inline tools/bin/hawq_ctl]: # get hawq_acl_type from local hawq_site.xml if connect master failed.

File: tools/bin/hawq_ctl [hunk 2]
```diff
@@ -835,6 +835,8 @@ class HawqStop:
...
+            # avoid cannot stop segments where master is down.
```
Comment (linwen) [inline tools/bin/hawq_ctl]: This line could be removed.

**Comment by linwen** [comment]
+1

**Comment by interma** [comment]
Thanks @ictmalili, already fixed.

**Comment by interma** [comment]
merged

**Pull request closed without merging**